learning/katas/go/Core Transforms/Side Output/Side Output/pkg/task/task.go

```diff
@@ -0,0 +1,30 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package task
+
+import (
+	"github.com/apache/beam/sdks/go/pkg/beam"
+)
+
+func ApplyTransform(s beam.Scope, input beam.PCollection) (beam.PCollection, beam.PCollection) {
+	return beam.ParDo2(s, func(element int, numBelow100, numAbove100 func(int)) {
+		if element <= 100 {
+			numBelow100(element)
+			return
+		}
+		numAbove100(element)
```

Comment:
While this works too, maybe you would want to consider using `else` statement.

Reply:
As written this is idiomatic Go.

Reply:
I will keep this unchanged then.  Thank you.